### DISCUSSION TITLE
How to set Drawer Nav header title

### DISCUSSION THREAD

**Discussion by stesvis** [react-navigation/react-navigation#10359] [Q&A]
Hello,
I have a **Tab** navigator inside a **Drawer** navigator, such as:

```
    <Drawer.Navigator
       initialRouteName={Routes.home}
       screenOptions={({ route }) => drawerNavigationOptions(route)}>
       <Drawer.Screen name={Routes.home} component={HomeTabNavigator} />
    </Drawer.Navigator>
```

I would like to set the `headerTitle` of the drawer navigator based on the tab selected.
Here's a screenshot to help understand: 
![image](https://user-images.githubusercontent.com/6062684/154383293-66989403-6591-4f5b-8b06-7eb8b9abdb40.png)
![image](https://user-images.githubusercontent.com/6062684/154382713-af0c408b-883d-4d6e-b2fd-af0804d6e5bb.png)

Basically when I select a tab, in this case **Deliveries**, in the header I would like to see **Deliveries** instead of **home**.
At the moment it doesn't matter on which tab i am, the title is always **home**.

How can I achieve that? It's probably a silly question, but I am coming from Xamarin so I am not too expert with react native.
Thanks

**Comment by raajnadar** [comment]
With react-navigation v6 this is done automatically by adding `headerShown` as `true`, for v5 you need to add the `headerShown` in the options of the screen or the `screenOptions` of the navigator.

https://reactnavigation.org/docs/drawer-navigator#headershown
